### ISSUE TITLE
Training Problem

### ISSUE THREAD

**Issue by wjc2400136** [krishnakanthnakka/NAT#1]
Dear author, I'd like to ask what Python files I should add in order to generate the `.pth` weight file that I've trained using the NAT method in your project. Could you please supplement relevant folders or Python files regarding this? Thank you for your reply!

**Comment by krishnakanthnakka** [comment]
Hi @wjc2400136 ,

I have used the code of my earlier [LTP](https://github.com/krishnakanthnakka/Transferable_Perturbations) and modified the loss.py script and generator.py script in that repo. 

In the LTP repo, please run the command, `bash scripts/imagenet/vgg16.sh`

I will try to integrate the training code here when I have time.
